.github/workflows/proto-buf-publisher.yml

```
@@ -4,10 +4,14 @@ name: Proto Buf Publishing - Action
 on:
   push:
     branches:
-      - master
+      - main
     paths:
       - 'proto/**'
 
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

nit: let's publish for RC versions as well, so that people can integrate/ test early
```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+.*" # Push events to matching v*, //i.e. v20.15.10, v0.27.0-rc1
```